### ISSUE TITLE
Problem

### ISSUE THREAD

**Issue by dclcs** [nywang16/Pixel2Mesh#65]
when i train the model , i meet the problem :
Caused by op u'Gather_12', defined at:
  File "/home/dcl/workspace/Pixel2Mesh/train.py", line 55, in <module>
    model = GCN(placeholders, logging=True)
  File "/home/dcl/workspace/Pixel2Mesh/p2m/models.py", line 126, in __init__
    self.build()
  File "/home/dcl/workspace/Pixel2Mesh/p2m/models.py", line 91, in build
    self._loss()
  File "/home/dcl/workspace/Pixel2Mesh/p2m/models.py", line 136, in _loss
    self.loss += .1*laplace_loss(self.inputs, self.output1, self.placeholders, 1)
  File "/home/dcl/workspace/Pixel2Mesh/p2m/losses.py", line 34, in laplace_loss
    lap1 = laplace_coord(pred1, placeholders, block_id)
  File "/home/dcl/workspace/Pixel2Mesh/p2m/losses.py", line 27, in laplace_coord
    laplace = tf.reduce_sum(tf.gather(vertex, indices), 1)
  File "/home/dcl/conda/envs/new_tf2/lib/python2.7/site-packages/tensorflow/python/ops/array_ops.py", line 2409, in gather
    validate_indices=validate_indices, name=name)
  File "/home/dcl/conda/envs/new_tf2/lib/python2.7/site-packages/tensorflow/python/ops/gen_array_ops.py", line 1219, in gather
    validate_indices=validate_indices, name=name)
  File "/home/dcl/conda/envs/new_tf2/lib/python2.7/site-packages/tensorflow/python/framework/op_def_library.py", line 767, in apply_op
    op_def=op_def)
  File "/home/dcl/conda/envs/new_tf2/lib/python2.7/site-packages/tensorflow/python/framework/ops.py", line 2630, in create_op
    original_op=self._default_original_op, op_def=op_def)
  File "/home/dcl/conda/envs/new_tf2/lib/python2.7/site-packages/tensorflow/python/framework/ops.py", line 1204, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

InvalidArgumentError (see above for traceback): indices[0,7] = -1 is not in [0, 157)
	 [[Node: Gather_12 = Gather[Tindices=DT_INT32, Tparams=DT_FLOAT, validate_indices=true, _device="/job:localhost/replica:0/task:0/cpu:0"](concat_2, strided_slice_15)]]

**Comment by qiao1025566574** [comment]
same problem.
`InvalidArgumentError (see above for traceback): flat indices[117, :] = [3, 7] does not index into param (shape: [7,7,512]).
         [[Node: graphprojection_1/GatherNd_15 = GatherNd[Tindices=DT_INT32, Tparams=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"](gcn/Squeeze_3, graphprojection_1/stack_15)]]`

**Comment by qiao1025566574** [comment]
See issue 22 [https://github.com/nywang16/Pixel2Mesh/issues/22](url)

**Comment by Sayan-m90** [comment]
any solution on this? i got a similar error

`2020-06-25 06:07:20.423197: W tensorflow/core/framework/op_kernel.cc:1651] OP_REQUIRES failed at gather_nd_op.cc:47 : Invalid argument: indices[1
17] = [2, 7] does not index into param shape [7,7,512]
2020-06-25 06:07:20.423594: W tensorflow/core/framework/op_kernel.cc:1651] OP_REQUIRES failed at gather_nd_op.cc:47 : Invalid argument: indices[1
17] = [3, 7] does not index into param shape [7,7,512]`

**Comment by Mitch1302** [comment]
Has anyone figured it out ? I get the same error. Please let me know if there is a solution for this.
